grails-plugin-controllers/src/main/groovy/org/grails/config/http/GrailsFilters.java

```diff
@@ -0,0 +1,48 @@
+package org.grails.config.http;
```

Comment:
I think it is better/more common to put the package below the license header.

Reply:
moved